tests/jsonParser.cpp

```diff
@@ -0,0 +1,127 @@
+/*++
```

Comment:
Do we still need this file?

Reply:
i removed copying and added the license header to mock.c and gtmain.cpp

i think the json file is needed for specifying a list of green bitstreams; so it is not needed as of now and it can be removed but that removal requires to modify gtmain.cpp too as there is no longer an input json file.

Reply:
Let's keep it for now in anticipation of tests that actually need JSON parsing.